package.json

```diff
@@ -107,12 +108,12 @@
     "rimraf": "5.0.5",
     "rollup": "4.9.4",
     "rollup-plugin-banner2": "1.2.2",
+    "sass": "^1.77.2",
     "sass-loader": "14.1.1",
     "storybook": "7.4.0",
     "tailwindcss": "3.4.1",
     "ts-jest": "29.1.1",
-    "typescript": "5.4.5",
-    "sass": "^1.77.2"
+    "typescript": "5.4.5"
```

Comment:
Same for typescript. Package version remains unchanged. Something changed the line it was on. Not sure what it was.